tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -688,6 +688,7 @@ export class WebGPUBackend extends KernelBackend {
     let currentOffset = 0;
     let preLength = 0;
     const offsets: number[] = [];
+    let maxAlignmentOfField = 4;
```

Comment:
Why not initialize it to `1` here? If there are int8 or fp16 members, will this code be also incorrect?